apps/web-portal-client/lib/features/examples/hooks/useSpecs.jsx

```diff
@@ -0,0 +1,27 @@
+import {useContext} from 'react'
+import {useDispatch, useSelector} from 'react-redux'
+import * as Specs from '../slices/specsSlice'
+import {AppContext} from '../../../context'
+import {selectSpecs, selectSpecsError, selectLoadingSpecs} from '../selectors'
+
+export function useSpecs() {
```

Comment:
Could you remove all specs hooks, selectors and reducers ?

Reply:
done